### DISCUSSION TITLE
Super Mario 64 PAL Game ID not working - PixelFX N64Digital (2.1.40a)

### DISCUSSION THREAD

**Discussion by ottelo9** [Polprzewodnikowy/N64FlashcartMenu#171] [Q&A]
I'm using the Game ID Feature in N64Digital HDMI Mod. The IDs for mostly all games are transmitted correctly via the joybus and N64Digital detects it. But not for Super Mario 64 PAL. I'm wondering it its a problem with N64Digital firmware or SC64? Does anyone else have this problem besides me? 
https://gitlab.com/pixelfx-public/n64-game-id

**Comment by networkfusion** [comment]
I am guessing that the ROM you are using is byteswapped so sending the wrong ID.

**Reply by ottelo9** [reply]
I checked my ROM and you‘re right. It was actually byteswapped. I converted it to Big Endian but still no luck with Game ID :(

![IMG_6029](https://github.com/user-attachments/assets/d5c8b4bf-30df-4f83-bee3-9e8623ecad55)

**Comment by networkfusion** [comment]
This should be fixed in the latest preview release.
